pyramid_oereb/lib/records/plr.py

```diff
@@ -180,13 +180,18 @@ def calculate(self, real_estate):
         tested_geometries = []
         inside = False
         for geometry in self.geometries:
-            if geometry.calculate(
-                    real_estate,
-                    self.min_length, self.min_area,
-                    self.length_unit, self.area_unit
-            ):
-                tested_geometries.append(geometry)
-                inside = True
+            try:
+                if geometry.calculate(
+                        real_estate,
+                        self.min_length, self.min_area,
+                        self.length_unit, self.area_unit
+                ):
+                    tested_geometries.append(geometry)
+                    inside = True
+            except Exception as ex:
+                log.info("PLR geometry check not possible, plr is {}, geometry.geom is {}"
```

Comment:
Why is this logged using `info()` instead of `error()`?

Reply:
Thanks for having a look @kdeininger , in fact, I am not really happy with this PR, I was trying to output ids of the involved objects, but did not succeed; the real estate is clear, but I am not sure from where to get a meaningful key regarding the geometry being intersected with the real estate geometry, to make it easier to look it up in the data source. Maybe you have a better idea.

Reply:
You could try to calculate a point inside the invalid geometry:
https://shapely.readthedocs.io/en/maint-1.6/manual.html#object.representative_point

Using the point and the topic of the invalid PLR, it should be possible to identify the feature.